apps/api/v2/src/modules/organizations/controllers/event-types/organizations-event-types.controller.ts

```diff
@@ -91,6 +94,30 @@ export class OrganizationsEventTypesController {
     };
   }
 
+  @Roles("TEAM_ADMIN")
+  @Post("/teams/:teamId/event-types/:eventTypeId/create-phone-call")
+  @UseGuards(ApiAuthGuard, IsOrgGuard, IsTeamInOrg, RolesGuard)
+  async createPhoneCall(
+    @Param("eventTypeId") eventTypeId: number,
```

Comment:
We also need to parse eventTypeId to number because `handleCreatePhoneCall` handler schema expects it to be a number.